### PR TITLE
Renaming trim to enforce_schema in parse_m3u

### PR DESCRIPTION
**Pull request by dineiar**

### CODE REVIEW

File: m3u_parser/m3u_parser.py [hunk 1]
```diff
@@ -452,18 +443,17 @@ def _get_m3u_content(self) -> str:
...
+            if "logo" in stream_info:
                 line += ' tvg-logo="{}"'.format(stream_info["logo"])
-            if stream_info.get("country", {}).get("code"):
+            if "country" in stream_info:
                 line += ' tvg-country="{}"'.format(stream_info["country"]["code"])
-            if stream_info.get("language", {}).get("name"):
-                line += ' tvg-language="{}"'.format(stream_info["language"]["name"])
-            if stream_info.get("category"):
+            if "language" in stream_info:
+                line += ' tvg-language="{}"'.format(stream_info["language"]["code"])
```
Comment (pawanpaudel93) [inline m3u_parser/m3u_parser.py:452]: Here I had kept the name and you replaced it with the code again ?? Any reason to do so ?? I followed the m3u files present here https://github.com/iptv-org/iptv/tree/master/channels.

Comment (dineiar) [reply m3u_parser/m3u_parser.py]: It was a mistake, sorry @pawanpaudel93. It should be fixed now (f34cd0d7221260bbbda52b655b531de95eddcbab).

**Comment by pawanpaudel93** [comment]
@dineiar If enforce_schema is set to True then for non-exising fields it will have None and if we save it to JSON it's fine but when we save it to m3u then it has "None" text for the non-existing fields. It's okay to have "None" in m3u ??

**Comment by dineiar** [comment]
@pawanpaudel93, this is a side-effect of [using `stream_info.get` that I mentioned](https://github.com/pawanpaudel93/m3u_parser/commit/4a414ad9cd143d4703d5e71c1d4d7d5a1885fdb7#r53810655) in you commit (4a414ad9cd143d4703d5e71c1d4d7d5a1885fdb7).

In this PR I fixed it by using `"key" in stream_info` instead of `stream_info.get("key", {})` around line 450 or m3u_parser.py, so it ignores the non-existing fields and doesn't fill "None".

**Comment by pawanpaudel93** [comment]
@dineiar Are you sure about what you are talking about if it doesn't fill None when saving to m3u file with enforce_schema=True with "key" in stream_info ??

**Comment by dineiar** [comment]
@pawanpaudel93, oh, I'm sorry, I was talking about using `enforce_schema=False`. I did not pay enough attention to your last comment.

I just fixed that issue in 319a918bc79ef2d47a0d07ec913ffaa28d3c37c6.

**Pull request merged**